code/modules/research/designs/mechfabricator_designs.dm

```diff
@@ -1148,6 +1148,50 @@
 		RND_CATEGORY_MECHFAB_CYBORG_MODULES + RND_SUBCATEGORY_MECHFAB_CYBORG_MODULES_JANITOR
 	)
 
+/datum/design/borg_upgrade_rollingtable
+	name = "Rolling Table Dock"
+	id = "borg_upgrade_rollingtable"
+	build_type = MECHFAB
+	build_path = /obj/item/borg/upgrade/rollingtable
+	materials = list(/datum/material/iron = 20000, /datum/material/titanium = 750) //steeper price than a regular rolling table, with some added titanium to make up for the relative rarity of regular rolling tables
+	construction_time = 40
```

Comment:
assuming these are in deciseconds, you should really patch this up to `4 SECONDS` instead... let me know if it's in something wacky like ticks and we can do something else.

ditto change this in all the other designs introduced in thsi PR.

Reply:
Works just dandy